docs/user-guide/khaas-101.md

```diff
@@ -0,0 +1,134 @@
+# KubeHound as a Service (KHaaS)
+
+KHaaS enables you to use KubeHound in a distributive way. It is being splitted in 2 main categories:
+
+* The ingestor stack which includes the `graphdb`, `datadb`, `UI` and `grpc endpoint`.
+* The collector (the kubehound binary) which will dump and send the k8s resources to the KHaaS `grpc endpoint`.
+
+[![](../images/khaas-architecture.png)](../images/khaas-architecture.png)
+
+!!! Note
```

Comment:
See [docs](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)
```suggestion
> [!NOTE]
```

Reply:
nop, this is for the wiki, it is not the same. Run the `make local-wiki`